### DISCUSSION TITLE
Indicate resolved/unresolved for comments

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#108389] [Mobile] [answered]
GitHub Mobile beta for Android

https://github.com/decidim/decidim/pull/12463
![Screenshot_20240215-080321.png](https://github.com/community/community/assets/2119212/2c585ecd-2997-4147-84f2-02bdfa8da662)

It'd be really helpful if the `6 comments` indicated how many were unresolved (and not outdated). Sometimes a PR (especially mine) will have dozens of comments, possibly added in different spans/spurts. On Mobile, it isn't fun to open a run to try to determine if there are any unresolved comments...

**Comment by HenSquared** [comment, marked as answer]
Thanks for the feedback! I've created a feature request for the team and we will discuss internally.
